xbmc/platform/android/CPUInfoAndroid.h

```diff
@@ -20,5 +25,8 @@ class CCPUInfoAndroid : public CCPUInfoPosix
 
   bool SupportsCPUUsage() const override { return false; }
   int GetUsedPercentage() override { return 0; }
-  float GetCPUFrequency() override { return 0; }
+  float GetCPUFrequency() override;
+
+private:
+  std::unique_ptr<CPosixFile> m_pPosixFile;
```

Comment:
Please don't use Hungarian notation.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-variables

Reply:
I'm sorry but I don't understand what you mean. Of course I can change the names of the variables if you wish. Can you be more specific and suggest a concrete change?

Reply:
hungarian notation prefixes the type (p for pointer in this example) to the variable name (PosixFile). What hes saying is dont do that, just do m_posixFile

Reply:
Thanks!  done